### ISSUE TITLE
can't reproduce your mAP

### ISSUE THREAD

**Issue by HaiyangLiu1997** [tensorboy/pytorch_Realtime_Multi-Person_Pose_Estimation#110]
hi,tensorboy, sorry for bother you again, 
today I have run your evaluation code, I use the pose_model you attach in readme, the acc has some proble.
By the way, I didn't change any other code except some path. so, how do you get the 0.65mAP, are there are some bugs in the current code?
![批注 2019-09-21 153733](https://user-images.githubusercontent.com/41906591/65369322-57e71080-dc86-11e9-999d-d10a9d0049f8.png)

**Comment by HaiyangLiu1997** [comment]
there are two problems, 
1. I download your pose_model and load it for training to confirm it loss, the average loss is 0.0030 for paf and 0.0022 for heatmap, it like some init weight or after one epoch weight,
2. I use one weight training by myself, it can get 0.58mAP in val2017, it also get the same 0.091mAP using your evaluate.py, so I think there are some bugs in the evaluation code.
from the current result by your code, obviously, you reproduce the openpose, and I think the real result may not so bad as I say, maybe there are some mistakes when you upload model. Could you release the mAP = 0.65 weight when you are free?

**Comment by HaiyangLiu1997** [comment]
supplement: in your old code,you using both train and val data to train, just using about 2K images marked by yourself to val, the result is overfitting, it get high mAP than openpose paper because the mini 2014val is in your training dataset, but it must be a low acc in test-dev leaderboard.

**Comment by wxywb** [comment]
hi, Haiyang, got exactly same result as 0.091.  By the way, I can get 0.353 after I switched preprocess from 'vgg' to 'rtpose'.
[]
DONE (t=6.55s).
Accumulating evaluation results...
DONE (t=0.10s).
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] = 0.353
 Average Precision  (AP) @[ IoU=0.50      | area=   all | maxDets= 20 ] = 0.613
 Average Precision  (AP) @[ IoU=0.75      | area=   all | maxDets= 20 ] = 0.344
 Average Precision  (AP) @[ IoU=0.50:0.95 | area=medium | maxDets= 20 ] = 0.420
 Average Precision  (AP) @[ IoU=0.50:0.95 | area= large | maxDets= 20 ] = 0.338
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=   all | maxDets= 20 ] = 0.539
 Average Recall     (AR) @[ IoU=0.50      | area=   all | maxDets= 20 ] = 0.756
 Average Recall     (AR) @[ IoU=0.75      | area=   all | maxDets= 20 ] = 0.564
 Average Recall     (AR) @[ IoU=0.50:0.95 | area=medium | maxDets= 20 ] = 0.451
 Average Recall     (AR) @[ IoU=0.50:0.95 | area= large | maxDets= 20 ] = 0.664

Could you give me some details about how to get 0.58 mAP in your experiment? the results seems not bad as the mAP indicated.

**Comment by chensonglu** [comment]
same problem
